src/NHSDPublicBrowseAcceptanceTests.Tests/Steps/Hooks.cs

```diff
@@ -19,6 +20,11 @@ public Hooks(UITest test, ScenarioContext context)
         [AfterScenario]
         public async Task AfterScenario()
         {
+            var screenshot = ((ITakesScreenshot) _test.Driver).GetScreenshot();
+            var fileName = _context.ScenarioInfo.Title;
+
+            screenshot.SaveAsFile(fileName + ".png", ScreenshotImageFormat.Png);
```

Comment:
are these going into a folder or just where we're running them from?
Is it worth having a config flag to turn this on and off?
How about a techdebt ticket to attach the screenshots to the reports in AzDo?

Reply:
I left this in as i'd been debugging, will be removing it shortly